xbmc/addons/kodi-dev-kit/include/kodi/addon-instance/pvr/Channels.h

```diff
@@ -41,7 +41,18 @@ class PVRChannel : public CStructHdl<PVRChannel, PVR_CHANNEL>
 
 public:
   /*! \cond PRIVATE */
-  PVRChannel() { memset(m_cStructure, 0, sizeof(PVR_CHANNEL)); }
+  PVRChannel()
+  {
+    m_cStructure->iUniqueId = 0;
+    m_cStructure->bIsRadio = 0;
+    m_cStructure->iChannelNumber = 0;
+    m_cStructure->iSubChannelNumber = 0;
+    m_cStructure->iEncryptionSystem = 0;
+    m_cStructure->bIsHidden = 0;
+    m_cStructure->bHasArchive = 0;
+    m_cStructure->iOrder = 0;
+    m_cStructure->iClientProviderUid = PVR_PROVIDER_INVALID_UID;
```

Comment:
Why this change? Now, whenever a new member is added this code needs to be adjusted.

Reply:
Because we require a default value other than 0. For each API type this is the pattern used. Although I guess I could memset and just change this value to 0, it would simplify adding new members.

Reply:
Done

Reply:
> I guess I could memset and just change set this value after, it would simplify adding new members.

Yes.